server/src/modules/reports/reports.controller.ts

```diff
@@ -8,12 +8,12 @@ import { LoggedInUser } from '@common/decorators/logged-in-user.decorator';
 @ApiTags('Reports')
 @ApiBearerAuth()
 export class ReportsController {
-    constructor(private readonly _reportsService: ReportsService) {}
+    constructor(private readonly reportsService: ReportsService) {}
 
     @Post()
     @ApiOperation({ summary: 'Create a report' })
     @ApiOkResponse({ type: ReportDto, description: 'The newly created report' })
-    createReport(@LoggedInUser('id') _submitter: number, @Query() _query: CreateReportDto): Promise<ReportDto> {
-        return void 0;
+    createReport(@LoggedInUser('id') submitter: number, @Body() query: CreateReportDto): Promise<ReportDto> {
+        return this.reportsService.createReport(submitter, query);
```

Comment:
You're right that this is a body not a query, so the variable should be `body` not `query`.